### DISCUSSION TITLE
Prompt library: Customizable categories

### DISCUSSION THREAD

**Discussion by Pfaufisch** [danny-avila/LibreChat#4013] [Feature Requests & Suggestions]
The prompt library is awesome! However, the pre-defined categories don't make a lot of sense for me.

It would be great if admins could customize the categories. Specifically, it should be possible to
- edit or delete existing categories
- create new categories
- per category, select an individual icon, color, and label

**Comment by danny-avila** [comment]
Sweet thanks for the feedback. There is an open issue, and I'm tracking this need! https://github.com/danny-avila/LibreChat/issues/3353
